en/installconfig-region.md

```diff
@@ -59,3 +59,22 @@ Check the log files for any errors:
 - `/var/log/maas/regiond.log`
 - `/var/log/maas/maas.log`
 - `/var/log/syslog`
+
+## Increasing the number of regiond daemon workers for improved performance
+
+!!! Note:
+    This functionality is available starting from MAAS 2.4.
+
+The MAAS Region Controller is a daemon collection of 4 workers that are in charge of handling all the internals of MAAS. The regiond workers handle the UI, API and the internal communication between Region and Rack controllers.
+
+Increasing the number of worker in larger environments where there are multiple rack controllers should increase the performance of the Region by allowing more workers to handle internal communication between the Region and the Rack controllers.
+
+This is possible by customizing ```regiond.conf (/etc/maas/regiond.conf)``` and setting ```num_workers```. For example:
+
+```
+[...]
+num_workers: 8
```

Comment:
It's best to set this to the same number of CPU's the system has. Each worker is single threaded, so setting it to 1 worker per CPU will provide the best performance.

Reply:
do we pin to a CPU? what if we have 80 cores? it doesn't make sense to do that....

Reply:
No it doesn't pin to a CPU, and yes 80 cores.... probably doesn't make sense, but 16 would. I wouldn't set it above 16. Everything with in reason of course... common sense.

I can go to an all you can eat buffet, but doesn't mean I am going to eat all I can possible fit in me until I explode. ;-)